app/middleware/grpc/client.js

```diff
@@ -14,18 +14,16 @@ var Buffer = require('buffer/').Buffer;
 export function client(address, port, cb) {
     var protoDescriptor = grpc.load('./app/api.proto');
     var walletrpc = protoDescriptor.walletrpc;
-    /*
     var certPath = path.join(process.env.HOME, '.dcrwallet', 'rpc.cert');
```

Comment:
I think it will fail like that as well.
That said, I think fixing that should be in a seperate issue and not part of the ssl enabling (since it is an issue that come from the dcrwallet doco and predate decrediton).